doc/user/content/integrations/cli/reference/profile.md

```diff
@@ -0,0 +1,179 @@
+---
+title: mz profile
+description: The `mz profile` command manages authentication profiles for `mz`.
+menu:
+  main:
+    parent: cli-reference
+    weight: 1
+---
+
+The `mz profile` command manages [authentication profiles] for `mz`.
+
+## `init`
+
+Initialize an authentication profile by exchanging your user account
+credentials for an app password.
+
+```shell
+mz profile init [options...]
+```
+
+### Arguments
+
+Argument                    | Environment variables | Description
+----------------------------|-----------------------|------------
+`--force`, `‑‑no-force`     |                       | Force reauthentication if the profile already exists.
+`--browser`, `‑‑no-browser` |                       | If set, open a web browser to authenticate. Otherwise, prompt for a username and password on the terminal.
+`--region=<REGION>`         |                       | Set the default region for the profile.
+`--profile=<PROFILE>`       | `MZ_PROFILE`          | Use the specified [authentication profile].
+
+
+### Examples
+
+```shell
+$ mz profile init --no-browser
+```
+```
+Email: remote@example.com
+Password: ...
+Successfully logged in.
+```
+
+## `list`, `ls`
+
+List available authentication profiles.
+
+```shell
+mz profile {list,ls} [options...]
+```
+
+### Examples
+
+```shell
+$ mz profile list
+```
+```
+Name
+------------
+development
+production
+staging
+```
+
+## `remove`, `rm`
+
+Remove an authentication profile.
+
+```shell
+mz profile {remove,rm} [options...]
+```
+
+### Arguments
+
+Argument              | Environment variables | Description
+----------------------|-----------------------|------------
+`--profile=<PROFILE>` | `MZ_PROFILE`          | Use the specified [authentication profile].
+
+### Examples
+
+Remove the `acme-corp` profile:
+
+```shell
+mz profile remove acme-corp
```

Comment:
To avoid doing it by mistake it should ask something like:
"_You are about to remove the profile `profile_name`, write y/n to continue_"

Reply:
That makes sense to me. To skip the prompt, should we support an option like `--yes`?